src/vmm/src/builder.rs

```diff
@@ -326,6 +327,9 @@ pub fn build_microvm_for_boot(
     #[allow(unused_mut)]
     let mut boot_cmdline = boot_config.cmdline.clone();
 
+    // Timestamp for measuring microVM boot duration.
+    let request_ts = TimestampUs::default();
+
```

Comment:
This is a unrelated change (basically reverts an unrelated commit), make sure to drop this change.